chart/values.yaml

```
@@ -150,7 +150,7 @@ images:
     pullPolicy: IfNotPresent
   gitSync:
     repository: k8s.gcr.io/git-sync
```

```suggestion
    repository: k8s.gcr.io/git-sync/git-sync
```

This also needs to be updated.

https://github.com/kubernetes/git-sync/releases/tag/v3.3.0

---

Nice catch, weird repetition of name.

Fixed in https://github.com/apache/airflow/pull/15606/commits/b1194d375bf09b955b030fc139a3cc739de27b17